File: platform-ui/src/displays/templates/TaskView.js
import React from "react";
import { withStyles } from '@material-ui/core/styles';
import AbstractTaskView from "../../../gen/displays/templates/AbstractTaskView";
import TaskViewNotifier from "../../../gen/displays/notifiers/TaskViewNotifier";
import TaskViewRequester from "../../../gen/displays/requesters/TaskViewRequester";
import DisplayFactory from 'alexandria-ui-elements/src/displays/DisplayFactory';
import { withSnackbar } from 'notistack';

const styles = theme => ({});

class TaskView extends AbstractTaskView {

	constructor(props) {
		super(props);
		this.notifier = new TaskViewNotifier(this);
		this.requester = new TaskViewRequester(this);
	};


}

export default withStyles(styles, { withTheme: true })(withSnackbar(TaskView));
DisplayFactory.register("TaskView", withStyles(styles, { withTheme: true })(withSnackbar(TaskView)));

File: platform/src/org/siani/goros/box/ui/displays/templates/TaskView.java
package org.siani.goros.box.ui.displays.templates;

import io.intino.alexandria.core.Box;
import org.siani.goros.box.ui.displays.notifiers.TaskViewNotifier;

public class TaskView<DN extends TaskViewNotifier, B extends Box> extends AbstractTaskView<DN, B> {

    public TaskView(B box) {
        super(box);
    }
    
}